addons/website_sale/controllers/main.py

```diff
@@ -1249,6 +1249,10 @@ def shop_payment_transaction(self, order_id, access_token, **kwargs):
         except AccessError:
             raise ValidationError("The access token is invalid.")
 
+        order = request.env['sale.order'].browse(order_id)
+        if order.state == 'cancel':
+            raise ValidationError('The order has been cancelled.')
```

Comment:
Canceled takes only one L in American english.
```suggestion
            raise ValidationError('The order has been canceled.')
```

Reply:
Updated.
It's automatically squashed upon merge, or should I squash commits now?

Reply:
No, you need to squash the commits locally and push the branch.

Reply:
Got it, thanks! It's done